### DISCUSSION TITLE
Error after Update

### DISCUSSION THREAD

**Discussion by DaPetra** [BartoszCichecki/LenovoLegionToolkit#1310] [General]
After updating to the latest version, an error message now appears on my Legion Pro 7 (Gen9) when I click on settings. After clicking OK, the program closes completely.

<img width="305" alt="2024-05-29 20_01_16-Error_LLT" src="https://github.com/BartoszCichecki/LenovoLegionToolkit/assets/92589051/d4b1c9c9-cb3c-453a-aae3-a377c36174f7">

**Comment by jonathanAB214** [comment]
I also have the same error pop up, Pro 7 Gen 9. I tried uninstalling + reboot + re-installing, same error.

**Reply by DaPetra** [reply]
I tried that too. I went back to the previous version

**Comment by BartoszCichecki** [comment]
Follow up here #1311.
